todo_app/app.py

```diff
@@ -1,22 +1,207 @@
 import os.path
-from flask import Flask,escape,request,Response,render_template, redirect
+import os
+import requests
+from flask import Flask,escape,request,Response,render_template, redirect,url_for
 import todo_app.data.session_items as session
 
 app = Flask(__name__)
-app.secret_key = "super secret key"
+key = os.getenv("TRELLO_KEY")
+token = os.getenv("TRELLO_TOKEN")
+headers = {
+   "Accept": "application/json"
+}
 
-@app.route('/add_item')
-def add_item():
-    title = request.args.get('name')
-    session.add_item(title)
-    return redirect("/")
+def get_tasks(s_id):
+    url = f"https://api.trello.com/1/lists/{s_id}/cards"
+    querystring = {"key": key, "token": token}
+    response = requests.request("GET", url, params=querystring)
+    data = [] 
+    response_json = response.json()
+    for a in response_json:
+        data.append({'id': a['id'], 'name': a['name']}) 
```

Comment:
Step 5 of the exercise asks you to create a class for the tasks instead of continuing to use dictionaries. Did you accidentally miss that?